connector/src/main/scala/com/microsoft/kusto/spark/utils/CslCommandsGenerator.scala

```diff
@@ -148,6 +133,10 @@ private[kusto] object CslCommandsGenerator {
     s""".alter table ${KustoQueryUtils.normalizeTableName(tmpTableName)} policy retention '{ "SoftDeletePeriod": "$period", "Recoverability":"${if (recoverable) "Enabled" else "Disabled"}" }' """
   }
 
+  def generateTableAlterAutoDeletePolicy(tmpTableName: String, expiryDate: String): String = {
```

Comment:
Why is the ExpiryDate (type of DateTime) parameter a String?

Reply:
will change